language_names/manual.py

```diff
@@ -23,13 +23,13 @@ def populate_database():
     c = conn.cursor()
     c.execute('PRAGMA foreign_keys=OFF;')
     c.execute('BEGIN TRANSACTION;')
-    insert_values(c, 'fixes.tsv', 'fixes')
-    insert_values(c, 'additions.tsv', 'additions')
-    insert_values(c, 'turkic_fixes.tsv', 'fixes')
-    insert_values(c, 'turkic_langNames.tsv', 'languageNames')
-    insert_values(c, 'scraped.tsv', 'languageNames')
-    insert_values(c, 'scraped-sil.tsv', 'languageNames')
-    insert_values(c, 'variants.tsv', 'languageNames')
+    insert_values(c, 'language_names/fixes.tsv', 'fixes')
```

Comment:
It seems like none of this is necessary if the filenames are passed through `argv`.

Reply:
Thanks, fixed it now.